src/index.html

```diff
@@ -5,27 +5,42 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link rel="stylesheet" href="./output.css">
     <title>Website project</title>
+    <script defer>
+        const loadComponents = (id, url, callback) => {
+            fetch(url)
+                .then(response => response.text())
+                .then(html => {
+                    document.getElementById(id).innerHTML = html;
+                    if (callback) callback();
+                });
+        }
+
+        window.onload = () => {
+            let loadedComponents = 0;
+            const totalComponents = 8;
+            
+            const componentLoaded = () => {
+                loadedComponents++;
+                if (loadedComponents === totalComponents) {
+                    service();       
+                }
+            };
+
+            loadComponents('header', 'sections/header.html', componentLoaded);
+            loadComponents('hero', 'sections/hero.html', componentLoaded);
+            loadComponents('about', 'sections/about.html', componentLoaded);
+            loadComponents('sticky', 'sections/sticky-yellow.html', componentLoaded);
+            loadComponents('services', 'sections/services.html', componentLoaded);
+            loadComponents('references', 'sections/references.html', componentLoaded);
```

Comment:
All of this are on dev branch I should not see it in this branch 
the rest all ok